include/dlaf/blas/enum_output.h

```diff
@@ -0,0 +1,81 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <ostream>
+#include "blas.hh"
+
+namespace blas {
+inline std::ostream& operator<<(std::ostream& out, const blas::Diag& diag) {
+  switch (diag) {
+    case blas::Diag::Unit:
+      out << "Unit";
+      break;
+    case blas::Diag::NonUnit:
+      out << "NonUnit";
+  }
```

Comment:
should we add a `break;` in this `case` as well?

```suggestion
  break;
  }
```